### PR TITLE
BC-7614 - Update Landing-Page (logged out) SC BB: Adjustment of support times

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description
Auf der Startseite Brandenburg https://brandenburg.cloud/ soll das Info-Feld zur "Allgemeine Anfragen & Support" angepasst werden.

## Links to Tickets or other pull requests

BC-7614
## Changes

- removing phone1, phone2 and online chat info on landing page brb

## Screenshots of UI changes

grafik

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -2246,9 +2246,6 @@
...
 				"welcome2": "Desde el 01.08.2021, la {{shortTitle}} se ha transferido a una infraestructura de enseñanza y aprendizaje interestatal sobre la base de un acuerdo administrativo entre los estados conjuntos de Brandenburgo, Baja Sajonia y Turingia, teniendo en cuenta los requisitos específicos de cada estado, y se está desarrollando conjuntamente.",
-				"phone1": "<i class=\"fa fa-phone\"></i> 03378 209 184 <a href=\"https://docs.dbildungscloud.de/display/SCDOK/Live-Formate\" target=\"_blank\">(Horario Oficial <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i>)</a>",
-				"phone2": "<i class=\"fa fa-phone\"></i> 03378 209 185 <a href=\"https://docs.dbildungscloud.de/display/SCDOK/Live-Formate\" target=\"_blank\">(Horario Oficial <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i>)</a>",
-				"onlineChat": "<i class=\"fa fa-comment\"></i> Online Chat <a href=\"https://docs.dbildungscloud.de/display/SCDOK/Live-Formate\" target=\"_blank\">(Sprechzeiten <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i>)</a>",
```
Comment (virgilchiriac) [inline locales/es.json:2251]: we should not remove this info, as this was intended to be temorary
I would instead simply comment out in the view

Comment (MartinSchuhmacher) [reply locales/es.json:2251]: you're right, I did not take focus on the "temporary" state, will adjust

**Review by Loki-Afro — CHANGES_REQUESTED** [review]
Do not comment out code,remove it if it's not for demonstration purposes

**Review by Loki-Afro — COMMENTED** [review]
History lives in git,not in comments

**Review by virgilchiriac — APPROVED** [review]

**Review by Loki-Afro — APPROVED** [review]

**Pull request merged**